randovania/games/fusion/logic_database/Sector 5 ARC.txt

```diff
@@ -383,41 +392,60 @@ Extra - room_id: [13]
   * Layers: default
   * L0 Hatch to Mimic Storage/Door to Frozen Tower
   * Extra - door_idx: (24,)
-  > Door to Groznyj Shaft
-      # TODO: varia / damag erun
+  > Bottom of Room
+      # TODO: dmg
       Varia Suit
-  > Door to Data Room
-      # TODO: varia / damage run
-      Space Jump
 
 > Door to Groznyj Shaft; Heals? False
   * Layers: default
   * L3 Hatch to Groznyj Shaft/Door to Frozen Tower
   * Extra - door_idx: (49,)
-  > Door to Mimic Storage
-      All of the following:
-          Space Jump or Can Freeze Enemies
-          # Damage Run
-          Varia Suit
-  > Other to Transmutation Trial
-      # TODO: varia / damage runs
-      Varia Suit and Can Bounce in Ball
+  > Bottom of Room
+      # TODO: dam
+      Varia Suit
 
 > Other to Transmutation Trial; Heals? False
   * Layers: default
   * Tunnel to Transmutation Trial/Other to Frozen Tower
   * Extra - door_idx: (79,)
-  > Door to Groznyj Shaft
-      # TODO: Varia / damag eruns
+  > Bottom of Room
+      # TODO: dmg
       Varia Suit
 
 > Door to Data Room; Heals? False
   * Layers: default
   * L3 Hatch to Data Room/Door to Frozen Tower
   * Extra - door_idx: (90,)
+  > Bottom of Room
+      # TODO: dmg
+      Varia Suit
+
+> Bottom of Room; Heals? False
+  * Layers: default
+  > Door to Mimic Storage
+      All of the following:
+          Any of the following:
+              Space Jump or Can Freeze Enemies
+              All of the following:
+                  Jump Extends (Advanced) and Can Single Walljump
+                  Any of the following:
+                      # w/ HJ: https://youtu.be/qIwNHrrM7uE
+                      Hi-Jump and Wall Jump (Intermediate)
+                      # NHJ: https://youtu.be/-GGGrkNAvF4
+                      Wall Jump (Advanced)
+          # TODO: dmg
+          Varia Suit
```

Comment:
i would prefer if youd change the requirements around here for varia to come first.